src/rdkafka_msgset_writer.c

```diff
@@ -948,7 +952,22 @@ rd_kafka_msgset_writer_compress_lz4 (rd_kafka_msgset_writer_t *msetw,
         return (err ? -1 : 0);
 }
 
-
+#if WITH_ZSTD
+/**
+ * @brief Compress messageset using ZSTD
+ */
+static int
+rd_kafka_msgset_writer_compress_zstd (rd_kafka_msgset_writer_t *msetw,
+                                     rd_slice_t *slice, struct iovec *ciov) {
+        rd_kafka_resp_err_t err;
+        int comp_level =
```

Comment:
micro nit: this temporary variable does not serve much purpose.

Reply:
it's the same as above, I think it's a bit neater as the rvalue is really long, but will remove it if you prefer that

Reply:
you can leave it 👍